tools/freqsap/freqSAP.xml

```diff
@@ -0,0 +1,133 @@
+<tool id="freqsap" name="freqSAP" version="1.0.0+galaxy0" profile="23.0" license="MIT">
+    <description>Get frequencies of single amino-acid polymorphisms based on nucleid-acid polymorphism for different populations from UniProt and DbSNP</description>
+
+    <creator>
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI" />
+    </creator>
+
+    <edam_topics>
+        <edam_topic>topic_0121</edam_topic>
+        <edam_topic>topic_3366</edam_topic>
+    </edam_topics>
+
+    <edam_operations>
+        <edam_operation>operation_3197</edam_operation>
+        <edam_operation>operation_2479</edam_operation>
+        <edam_operation>operation_2422</edam_operation>
+    </edam_operations>
+
```

Comment:
can you please add https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files-include